src/debug/ee/debugger.cpp

```diff
@@ -15136,7 +15136,8 @@ HRESULT Debugger::InitAppDomainIPC(void)
     } hEnsureCleanup(this);
 
     DWORD dwStrLen = 0;
-    WCHAR szExeName[MAX_LONGPATH];
+    PathString szExeNamePathString;
+    WCHAR * szExeName = szExeNamePathString.OpenUnicodeBuffer(MAX_LONGPATH);
```

Comment:
The whole buffer is always either on stack or on heap. It is on stack only if requested size <= reserved size - it will never be the case here because of requested size = MAX_LONGPATH and reserved size = 512.